internal/jobs/twitter.go

```diff
@@ -1182,24 +1311,38 @@ func (ts *TwitterScraper) ExecuteJob(j types.Job) (types.JobResult, error) {
 		return types.JobResult{Error: "job result data is empty"}, fmt.Errorf("job result data is empty")
 	}
 
-	// Unmarshal result to typed structure
-	var results []*teetypes.TweetResult
-	if err := jobResult.Unmarshal(&results); err != nil {
-		logrus.Errorf("Error while unmarshalling job result for job ID %s, type %s: %v", j.UUID, j.Type, err)
-		return types.JobResult{Error: "error unmarshalling job result for final validation and result length check"}, err
-	}
+	// Check if this is a non-tweet operation that doesn't return tweet results
+	isNonTweetOperation := strings.ToLower(jobArgs.QueryType) == "searchbyprofile" ||
+		strings.ToLower(jobArgs.QueryType) == "searchfollowers" ||
+		strings.ToLower(jobArgs.QueryType) == "getretweeters" ||
+		strings.ToLower(jobArgs.QueryType) == "getprofilebyid" ||
+		strings.ToLower(jobArgs.QueryType) == "getbyid" ||
+		strings.ToLower(jobArgs.QueryType) == "getspace" ||
+		strings.ToLower(jobArgs.QueryType) == "gettrends" ||
+		strings.ToLower(jobArgs.QueryType) == "getfollowing" ||
+		strings.ToLower(jobArgs.QueryType) == "getfollowers"
```

Comment:
how about declaring a global slice with all the values (structured), and using `slices.Contains` here? At the very least, store `strings.ToLower(...) in a variable so we're not calling it over and over again.